config/settings.yml

```diff
@@ -196,6 +196,22 @@ modules_appeals_api:
   notice_of_disagreement_pii_expunge_enabled: true
   notice_of_disagreement_updater_enabled: true
   reports:
+    daily_decision_review:
+      enabled: false
+      recipients:
+        - drew.fisher@adhocteam.us
+        - jack.schuss@oddball.io
+        - kelly@adhocteam.us
+        - laura.trager@adhocteam.us
+        - nathan.wright@oddball.io
+    weekly_decision_review:
+      enabled: false
+      recipients:
+        - drew.fisher@adhocteam.us
+        - jack.schuss@oddball.io
+        - kelly@adhocteam.us
+        - laura.trager@adhocteam.us
+        - nathan.wright@oddball.io
     decision_review:
```

Comment:
reports:
    decision_review:
      enabled: true
      recipients:
        
    daily_error:
      enabled: true
      recipients:

Reply:
I think the above is the correct settings namespacing... (at least that's what it is in production.) Are you changing it?